usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -598,7 +598,8 @@ static CK_RV get_ep11_target_for_apqn(uint_32 adapter, uint_32 domain,
                                       target_t *target, uint64_t flags);
 static void free_ep11_target_for_apqn(target_t target);
 static CK_RV update_ep11_attrs_from_blob(STDLL_TokData_t *tokdata,
-                                         SESSION * session, TEMPLATE *tmpl);
+                                         SESSION * session, TEMPLATE *tmpl,
```

Comment:
Syle: please make this `SESSION *session`. I know it was that before, but while you changing this line, lets correct it.